view/frontend/templates/analytics/initialize.phtml

```diff
@@ -2,15 +2,23 @@
     <script text="text/javascript">
         var _learnq = _learnq || [];
         _learnq.push(['account', '<?php echo $this->getPublicApiKey(); ?>']);
-        (function () { var b = document.createElement('script'); b.type = 'text/javascript'; b.async = true; b.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'a.klaviyo.com/media/js/analytics/analytics.js'; var a = document.getElementsByTagName('script')[0]; a.parentNode.insertBefore(b, a); })();
+        window.addEventListener('load', function(event) {
+            (function () {
+                var b = document.createElement('script');
+                b.type = 'text/javascript';
+                b.async = true;
+                b.src = ('https:' == document.location.protocol ? 'https://' : 'http://') + 'a.klaviyo.com/media/js/analytics/analytics.js';
```

Comment:
It is possible to include http at all times, it will be fine too.

Reply:
Probably should leave it as is in the event that someone decides to not use https for some strange reason

Reply:
You are allowed to connect to https at all times, but provides latency for security handshakes.